README.md

```diff
@@ -1,4 +1,27 @@
 This is an experimental pkcs11 token written in rust
 
-To make the module functional in fips mode you need to run hmacify.sh after the
-build.
+# Dependencies
+
+ * rustc
+ * openssl dependencies
+
+# Setup
+
+First after cloning, we need to pull and update openssl submodule:
+
+$ git submodule init
+$ git submodule update
+
+Build the rust project:
+
+$ cargo build
```

Comment:
we may want to specify that --features fips is needed if the fips version is to be built ?

Reply:
I did not touch it after we talked this week that it might make sense to use FIPS by default. If we will not go that direction, the fips feature should be indeed separately documented.